src/tests/test_blindsig.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+"""
+Test for ECC blind signatures
+"""
+import os
+import unittest
+
+from src.pyelliptic.eccblind import ECCBlind
```

Comment:
`from pybitmessage.pyelliptic.eccblind import ECCBlind`

Reply:
This breaks the test.

Reply:
It depends on how you run it.

Reply:
Travis CI should be OK, because I used such imports before: https://github.com/Bitmessage/PyBitmessage/blob/v0.6/src/tests/test_config.py#L8